modules/programs/mcfly.nix

```diff
@@ -30,6 +48,8 @@ in {
       '';
     };
 
+    mcfly-fzf.enable = mkEnableOption "McFly fzf integration";
```

Comment:
I think you could omit the `mcfly-` from the option name, it seems a bit redundant.
```suggestion
    fzf.enable = mkEnableOption "McFly fzf integration";
```

Reply:
Yeah it does, just went ahead and added the change